x-pack/plugins/ml/public/jobs/new_job/simple/components/watcher/create_watch_view.js

```diff
@@ -0,0 +1,208 @@
+    this.config = mlCreateWatchService.config;
+
+    this.state = {
+      jobId: this.props.jobId,
+      bucketSpan: this.props.bucketSpan,
+      interval: this.config.interval,
+      threshold: this.config.threshold,
+      includeEmail: this.config.emailIncluded,
+      email: this.config.email,
+      emailEnabled: false,
+      status: null,
+      watchAlreadyExists: false,
+    };
+
+  }
+
+  componentDidMount() {
+    // make the interval 2 times the bucket span
+    if (this.state.bucketSpan) {
+      const intervalObject = parseInterval(this.state.bucketSpan);
+      let bs = intervalObject.asMinutes() * 2;
+      if (bs < 1) {
+        bs = 1;
+      }
+
+      const interval = `${bs}m`;
+      this.setState({ interval }, () => {
+        this.config.interval = interval;
+      });
+    }
+
+    // load elasticsearch settings to see if email has been configured
+    ml.getNotificationSettings().then((resp) => {
+      if (has(resp, 'defaults.xpack.notification.email')) {
+        this.setState({ emailEnabled: true });
+      }
+    });
+
+    mlCreateWatchService.loadWatch(this.state.jobId)
+      .then(() => {
+        this.setState({ watchAlreadyExists: true });
+      })
+      .catch(() => {
+        this.setState({ watchAlreadyExists: false });
+      });
+  }
+
+  onThresholdChange = (threshold) => {
+    this.setState({ threshold }, () => {
+      this.config.threshold = threshold;
+    });
+  }
+
+  onIntervalChange = (e) => {
+    const interval = e.target.value;
+    this.setState({ interval }, () => {
+      this.config.interval = interval;
+    });
+  }
+
+  onIncludeEmailChanged = (e) => {
+    const includeEmail = e.target.checked;
+    this.setState({ includeEmail }, () => {
+      this.config.includeEmail = includeEmail;
+    });
+  }
+
+  onEmailChange = (e) => {
+    const email = e.target.value;
+    this.setState({ email }, () => {
+      this.config.email = email;
+    });
+  }
+
+  render() {
+    const mlSelectSeverityService = {
+      state: {
+        set: (name, threshold) => {
+          this.onThresholdChange(threshold);
+          return {
+            changed: () => {}
+          };
+        },
+        get: () => {
+          return this.config.threshold;
+        },
+      }
+    };
+    const { status } = this.state;
+
+    if (status === null || status === STATUS.SAVING || status === STATUS.SAVE_FAILED) {
+      return (
+        <div className="create-watch">
+          <div className="form-group form-group-flex">
+            <div className="sub-form-group">
+              <div>
+                <label
+                  htmlFor="selectInterval"
+                  className="euiFormLabel"
+                >
+                  Time range
+                </label>
+              </div>
+              Now - <EuiFieldText
+                value={this.state.interval}
+                onChange={this.onIntervalChange}
+                aria-label="Use aria labels when no actual label is in use"
```

Comment:
Did you mean to leave this `aria-label` value in like this? Shouldn't it say something like `Set to now`?